syntax = "proto3";
package Avalon.Network.Packets.Handshake;

message CRequestServerInfoPacket {
   int32 ClientVersion = 1;
}
